src/components/navigation-header/style.module.css

```diff
@@ -33,20 +33,36 @@
   list-style: none;
   margin-bottom: 0;
   margin-left: 0;
-  margin-right: 56px;
+  margin-right: 72px;
   margin-top: 0;
 
   & li {
-    padding: 8px 12px;
+    padding: 4px;
 
     & a {
-      color: var(--navigation-header-link-color);
+      border: 1px solid transparent;
+      border-radius: 5px;
+      color: var(--token-color-palette-neutral-400);
+      display: block;
       font-size: 14px;
-      line-height: 16px;
+      font-weight: 500;
+      line-height: 20px;
+      padding: 3px 8px;
```

Comment:
nit: most of our spacing is base 4, I wonder if it'd be feasible to adjust this to `padding: 4px 8px` 💭 

No big deal if not, just an observation!

Reply:
So this is because of some math I had to do to make sure that a focus border doesn't add layout shift. I've had to do this in a lot of places across my PRs, so I think I'll do a follow-up PR for cleaning these things up with variables & `calc` to make it explicit that some math had to happen. What do you think?

Reply:
That's fine! A comment explaining why we're using odd values could be helpful too, if you don't wanna introduce a new approach 👍

Reply:
I took a combined approach and added a TODO with a reason. 😊